src/client.ts

```diff
@@ -59,7 +59,7 @@ class Client {
     public static MARKETPAY_ACCOUNT_API_VERSION: string = "v4";
     public static MARKETPAY_FUND_API_VERSION: string = "v3";
     public static MARKETPAY_NOTIFICATION_API_VERSION: string = "v1";
-    public static LIB_NAME: string = "adyen-node-api-library";
+    public static LIB_NAME: string = "api-library";
```

Comment:
let's keep this adyen-node-api-library so we have consistent data in the dashboard. and here we do not know automatically it is node library.